### PR TITLE
Get Block Range API 

### PR DESCRIPTION
**Pull request by Rishi556**
Allows for getting information on blocks in a range with a single call. 

This is a very heavy api and should not be enabled on public nodes. To make sure this doesn't happen, a config feature has been added to allow disabling certain RPC methods. The default config disables the new method.

### CODE REVIEW

File: plugins/JsonRPCServer.js [hunk 1]
```diff
@@ -106,6 +106,30 @@ function blockchainRPC() {
...
+          callback({
+            code: 400,
+            message: 'missing or wrong parameters: blockNumber is required',
+          }, null);
+          return;
+        }
+        if (!Number.isInteger(count) || count > 1000) {
+          callback({
+            code: 400,
+            message: 'missing or wrong parameters: count is required',
```
Comment (bt-cryptomancer) [inline plugins/JsonRPCServer.js:123]: If count > 1000 can you have a different error message returned?

File: config.json [hunk 2]
```diff
@@ -22,7 +22,11 @@
     "rpcConfig" : {
         "maxLimit" : 1000,
         "maxOffset" : -1,
-        "logRequests" : false
+        "logRequests" : false,
+        "disabledMethods" : {
```
Comment (bt-cryptomancer) [inline config.json:26]: Can you have the `disabledMethods` info added to the `getStatus` API?  That way applications could detect which APIs are not supported by a given node, without having to guess.

**Review by bt-cryptomancer — CHANGES_REQUESTED** [review]
Mostly looks good, just a couple things that can be improved on (see comments below).

File: config.json (hunk 2 above)
Comment (Rishi556) [reply config.json:26]: Completed in https://github.com/hive-engine/hivesmartcontracts/pull/16/commits/91434980045f0a871b812d99b28fd671c08f4cc4

File: plugins/JsonRPCServer.js (hunk 1 above)
Comment (Rishi556) [reply plugins/JsonRPCServer.js:126]: Completed in https://github.com/hive-engine/hivesmartcontracts/pull/16/commits/91434980045f0a871b812d99b28fd671c08f4cc4

**Review by bt-cryptomancer — CHANGES_REQUESTED** [review]
One more thing: please make sure that the `getStatus` API itself can never be disabled.  It should always be possible to check on a node's status.

**Comment by Rishi556** [comment]
> One more thing: please make sure that the `getStatus` API itself can never be disabled. It should always be possible to check on a node's status.

Resolved in https://github.com/hive-engine/hivesmartcontracts/pull/16/commits/7c2b045e47c6ccb60b92c2070ff185e5563a1486. 

While the operator can still place it in the blocklist, the block of `getStatus` won't be respected.

**Review by bt-cryptomancer — APPROVED** [review]
Everything looks good now, thanks!

**Pull request merged**